kodilla-advanced-tests/src/test/java/com/kodilla/parametrized_tests/homework/GamblingMachineTestSuite.java

```diff
@@ -15,7 +15,7 @@
     @ParameterizedTest
     @CsvFileSource(resources = "/GoodNumbers.csv")
     public void shouldCheckWinsWork(String numbers) throws InvalidNumbersException {
-        String[] parsedNumbers = numbers.split("4,9,43,25,37,38");
+        String[] parsedNumbers = numbers.split(",");
```

Comment:
A po co sa linie 23-31 oraz 45-55? Obiekt GamblingMachine powinien byc utworzony po petli for.